Dev

nlukic97 opened the pull request:
Added the requested changes for the Profile model, and it's methods.

nlukic97 commented:
Created a model association between **User** and **Profile** in the form of a **OneToOne** relationship - will update if necessary.

https://github.com/nlukic97/uteam-api/blob/db0bebcecca85da6f580460aefe977db10d10e1b/ts/models/User.ts#L50-L52

vuxor commented on ts/controllers/Profile.controller.ts:
```diff
@@ -0,0 +1,206 @@
+        // constructing the profile
+        const profile = await Profile.build(submitData)
+        
+        profile.save().then(()=>{
```
You used async/await previously in this function. Try to convert promise.

nlukic97 replied:
https://github.com/nlukic97/uteam-api/blob/a2ce8efe4c1a75b7089404d55f58539788439279/ts/controllers/Profile.controller.ts#L72

Fixed - removed await and used the "create" method which returns a promise.

nlukic97 commented:
Added README instructions on setting up ACCESS_TOKEN_SECRET in .env needed for JWT token creation. 
Also added a check for this .env variable in **app.ts**:
https://github.com/nlukic97/uteam-api/blob/815edc0a23b61e4e5a92a40721e771cd21feba2d/ts/app.ts#L8-L9

Without this check, the server can run without the previously mentioned .env variable, but will eventually crash when trying to create a JWT token for a newly registered user.
